morphic.js

```diff
@@ -10224,14 +10224,12 @@ WorldMorph.prototype.initVirtualKeyboard = function () {
                 myself.keyboardReceiver.processKeyDown(event);
             }
             // supress backspace override
-            if (event.keyIdentifier === 'U+0008' ||
-                    event.keyIdentifier === 'Backspace') {
+            if (event.keyCode === 8) {
```

Comment:
I couldn't believe it either, because it's been a while since I touched such low-level stuff, but it looks like the support for `key` and `keyIdentifier` properties is vastly inconsistent among browsers. Even though they were supposed to replace `keyCode`, the latter is still the simplest way to solve it.